dompurify/dompurify.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for DOM Purify
+// Project: https://github.com/cure53/DOMPurify
+// Definitions by: Dave Taylor <http://davetayls.me>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+interface IDOMPurify {
+  sanitize(s:string):string;
+  addHook(hook:string, cb:(currentNode:Element, data:any, config:any) => Element):void;
+}
+
+declare var DOMPurify:IDOMPurify;
```

Comment:
@vvakame https://github.com/borisyankov/DefinitelyTyped/blob/master/lodash/lodash.d.ts#L6 Lodash does the same. 

It avoids having to do `(<any>window).xxx`.